### PR TITLE
LOOP-2534 Refactor onboarding and support manager plugins

### PR DESCRIPTION
**Pull request by darinkrauss**
- https://tidepool.atlassian.net/browse/LOOP-2534
- https://tidepool.atlassian.net/browse/LOOP-2955
- Create initial TidepoolOnboarding project and targets
- Move onboarding from TidepoolService plugin/repo
- Use PrescriptionReviewUICoordinator as temporary onboarding

Note: Needs TWO reviewers.

### CODE REVIEW

**Comment by darinkrauss** [comment]
Repository is now public.

File: TidepoolOnboardingKitUI/View Controllers/PrescriptionReviewUICoordinator.swift [hunk 1]
```diff
@@ -0,0 +1,376 @@
+//
+//  PrescriptionReviewUICoordinator.swift
+//  TidepoolOnboardingKitUI
+//
+//  Created by Anna Quinlan on 6/18/20.
+//  Copyright © 2020 Tidepool Project. All rights reserved.
+//
+
+import os.log
```
Comment (nhamming) [inline TidepoolOnboardingKitUI/View Controllers/PrescriptionReviewUICoordinator.swift:9]: this looks like a cut and paste of the current code. If that isn't the case, let me know and I'll look at it closely.

**Review by nhamming — APPROVED** [review]
Much of this is either project template like code or re-used code. LGTM.

File: TidepoolOnboardingKitUI/View Controllers/PrescriptionReviewUICoordinator.swift (hunk 1 above)
Comment (darinkrauss) [reply TidepoolOnboardingKitUI/View Controllers/PrescriptionReviewUICoordinator.swift:9]: Yes, this was mostly just moving this file from the `TidepoolService` repo to this repo. There is some temporary "glue" code (all of the delegates at the top) to connect it into the onboarding workflow, but most of that has gone away with the next set of PRs anyhow. So, yes, mostly cut and paste.

Comment (rickpasetto) [reply TidepoolOnboardingKitUI/View Controllers/PrescriptionReviewUICoordinator.swift:9]: I ditto Nate's comment.

File: TidepoolOnboardingKitUI/View Models/PrescriptionReviewViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,73 @@
+//
+//  PrescriptionCodeEntryViewModel.swift
```
Comment (rickpasetto) [inline TidepoolOnboardingKitUI/View Models/PrescriptionReviewViewModel.swift:2]: Ditto here.  In fact, I'm going to assume all of the code from TidepoolService was just copy/pasted here.

**Review by rickpasetto — APPROVED** [review]
LGTM 🦖

**Review by ps2 — APPROVED** [review]
LGTM!

**Pull request merged**